etc/jenkins/pr_verify.groovy

```diff
@@ -159,9 +159,7 @@ spec:
                     steps {
                         container('el-build') {
                             sh """
-                                /opt/bin/mongo-start.sh
-                                mvn -B -V verify -pl :org.eclipse.persistence.nosql -P staging,mongodb
-                                /opt/bin/mongo-stop.sh                                
+                                mvn -B -V verify -pl :org.eclipse.persistence.jpa.testapps.nosql.mongo -P staging
```

Comment:
the step can be completely removed, tests run within regular build already

Reply:
OK mentioned `stage` was removed.